rpms/nitrokey-app/nitrokey-app.spec

```diff
@@ -0,0 +1,82 @@
+%global _icondir %{_datadir}/icons/hicolor
+
+Name:		nitrokey-app
+Version:	0.5.1
+Release:	1%{?dist}
+Summary:	Nitrokey Configuration tool
+
+Group:		Applications/System
+License:	GPLv3
+URL:		https://github.com/Nitrokey/nitrokey-app
+#Source0:	%{url}/%{_commit}/%{repo}-%{_shortcommit}.tar.gz
+Source0:	%{url}/archive/v%{version}.tar.gz
+Source1:	https://www.nitrokey.com/sites/default/files/40-nitrokey.rules
```

Comment:
https://github.com/Nitrokey/nitrokey-app/blob/master/data/40-nitrokey.rules
The file is already here.